v-next/example-project/package.json

```
@@ -17,7 +17,8 @@
     "lint:fix": "pnpm prettier --write",
     "prettier": "prettier \"**/*.{ts,js,md,json}\"",
     "build": "tsc --build .",
-    "clean": "rimraf dist"
+    "clean": "rimraf dist",
+    "test": "hardhat3 test node && hardhat3 test mocha"
```

Like suggested in the meeting, we can take inspiration from https://github.com/NomicFoundation/hardhat/blob/v-next/v-next/hardhat/package.json#L49

So something along the lines of the following should work:

```suggestion
    "test": "hardhat3 test node && hardhat3 test mocha",
    "pretest": "pnpm build && pnpm install"
```

---

It worked! should be good to go now